py/README.md

```diff
@@ -1,16 +1,29 @@
 Python user acceptance tests
 ==========
 
-In order to generate tests:
-* prepare nosetests venv: `make nose-env`
+This document describes how to run them.
 
-* generate python client for the API: `make swagger`
+#### Python enviroment
+
+Python tests are to be run on Python `2.7` and this is a prerequisite. Tests
+dependencies are installed in an virtual enviroment. To install deps:
+```
+make python-env
```

Comment:
Shall [CI](https://github.com/aeternity/epoch/blob/fb95e6724f5a608696b4a1b76b82c2a83b61c337/.circleci/config.yml#L304) be simplified too?